prime-dotnet-webapi/HttpClients/Org Book API/IOrgBookClient.cs

```diff
@@ -0,0 +1,11 @@
+using System.Threading.Tasks;
+
+using Prime.HttpClients.PharmanetCollegeApiDefinitions;
+
+namespace Prime.HttpClients
+{
+    public interface IOrgBookClient
+    {
+        Task<string> GetOrgBookSearchRecordAsync(string orgName);
```

Comment:
Would `GetOrgBookRegistrationIdAsync` be more accurate?

Reply:
Update as suggested.